pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/QuestionDao.java

```diff
@@ -661,15 +661,14 @@ default CompositeQuestion getCompositeQuestion(QuestionDto dto,
                                                    String activityInstanceGuid,
                                                    List<Long> answerIds,
                                                    List<Rule> untypedRules,
-                                                   boolean retrieveAnswers,
                                                    long langCodeId) {
         CompositeQuestionDto compositeQuestionDto = getJdbiCompositeQuestion()
                 .findDtoByQuestion(dto)
                 .orElseThrow(() -> new DaoException("Could not find composite question using question id " + dto.getId()));
 
         List<Question> childQuestions = compositeQuestionDto.getChildQuestions().stream()
                 .map(childQuestionDto ->
-                        getQuestionByActivityInstanceAndDto(childQuestionDto, activityInstanceGuid, retrieveAnswers, langCodeId))
+                        getQuestionByActivityInstanceAndDto(childQuestionDto, activityInstanceGuid, false, langCodeId))
```

Comment:
We have already fetched the parent composite answer, so no need to fetch the child answers.